nl_server/gcs.py

```diff
@@ -32,5 +34,59 @@ def download_embeddings(embeddings_file: str) -> str:
   return local_embeddings_path
 
 
+def local_folder() -> str:
+  return TEMP_DIR
```

Comment:
nit:  Is there a case where we use anything other than TEMP_DIR?  So wonder why add a layer of indirection and arg to download_model_* APIs?

Reply:
updated